general-superstaq/general_superstaq/superstaq_client.py

```diff
@@ -213,22 +218,27 @@ def _accept_terms_of_use(self, user_input: str) -> str:
         """Makes a POST request to Superstaq API to confirm acceptance of terms of use.
 
         Args:
-            user_input: user's response to prompt for acceptance of TOU. Server accepts YES
+            user_input: The user's response to prompt for acceptance of TOU. Server accepts YES.
 
         Returns:
             String with success message.
         """
         return self.post_request("/accept_terms_of_use", {"user_input": user_input})
 
     def get_targets(self) -> Dict[str, Dict[str, List[str]]]:
-        """Makes a GET request to Superstaq API to get a list of available targets."""
+        """Makes a GET request to the Superstaq API to get a list of available, unavailable,
+        and retired targets.
```

Comment:
```suggestion
        """Makes a GET request to the Superstaq API.
        
        Gets a list of available, unavailable, and retired targets.
```
(or something like that)

Reply:
Done in [f98da3c](https://github.com/Infleqtion/client-superstaq/pull/590/commits/f98da3c10266d3819e15851de8e7e4c78304640f)